ghcr.io/cachix/devenv > v1.3.1 broken when used at GitHub Codespaces

If I use never image than

```
  "image": "ghcr.io/cachix/devenv:v1.3.1",
```

e.g

```
  "image": "ghcr.io/cachix/devenv:v1.4",
```

or

```
  "image": "ghcr.io/cachix/devenv:main",
```

at GitHub Codespaces, the codespace build fails with

```
=================================================================================
2025-03-26 05:12:27.883Z: Running blocking commands...
2025-03-26 05:12:27.943Z: $ devcontainer up --id-label Type=codespaces --workspace-folder /var/lib/docker/codespacemount/workspace/2025.ploneconf.org --mount type=bind,source=/.codespaces/agent/mount/cache,target=/vscode --user-data-folder /var/lib/docker/codespacemount/.persistedshare --container-data-folder .vscode-remote/data/Machine --container-system-data-folder /var/vscode-remote --log-level trace --log-format json --update-remote-user-uid-default never --mount-workspace-git-root false --omit-config-remote-env-from-metadata --skip-non-blocking-commands --expect-existing-container --config "/var/lib/docker/codespacemount/workspace/2025.ploneconf.org/.devcontainer.json" --override-config /root/.codespaces/shared/merged_devcontainer.json --default-user-env-probe loginInteractiveShell --container-session-data-folder /workspaces/.codespaces/.persistedshare/devcontainers-cli/cache --secrets-file /root/.codespaces/shared/user-secrets-envs.json
2025-03-26 05:12:28.222Z: @devcontainers/cli 0.68.0. Node.js v18.20.6. linux 6.8.0-1021-azure x64.
2025-03-26 05:12:28.485Z: Running the updateContentCommand from devcontainer.json...

2025-03-26 05:12:28.485Z: make codespace-install
2025-03-26 05:12:28.670Z: + dirname /workspaces/2025.ploneconf.org/Makefile
2025-03-26 05:12:28.672Z: + echo CODESPACE_NAME=********
2025-03-26 05:12:28.672Z: + echo GITHUB_CODESPACES_PORT_FORWARDING_DOMAIN=********
2025-03-26 05:12:28.673Z: + devenv shell -- echo Ok.
2025-03-26 05:12:28.730Z: • Building shell ...
2025-03-26 05:12:51.600Z: • Using Cachix: devenv
Trusting devenv.cachix.org on first use with the public key devenv.cachix.org-1:w1cLUi8dv3hnoSPGAuibQv+f9TZLr6cv/Hm9XgU50cw=
2025-03-26 05:13:11.411Z: error: suspicious ownership or permission on '/nix/store/mrhn5zmcq7yg91zsgvs4mjgg4gbfzc7p-Caddyfile' for output 'out'; rejecting this build output
error: 1 dependencies of derivation '/nix/store/z29cr8q3198yypsc01bd2rnzjj1ksaja-formattedCaddyFile.drv' failed to build
2025-03-26 05:13:11.478Z: error: 1 dependencies of derivation '/nix/store/7fwzji67vhwwip2skll87fpwbpswyvaf-caddy-config-adapted.json.drv' failed to build
2025-03-26 05:13:11.551Z: error: suspicious ownership or permission on '/nix/store/x53v0gaimxmpy3qwaplj7fmcf8b66gnw-backend' for output 'out'; rejecting this build output
2025-03-26 05:13:11.792Z: error: suspicious ownership or permission on '/nix/store/hglasf451lafcihjgbg1arwsv86mzk6n-bash-backend-install' for output 'out'; rejecting this build output
2025-03-26 05:13:11.974Z: error: suspicious ownership or permission on '/nix/store/7v9d29dh5vkdmiplgky0a049dkqn2am2-bash-frontend-install' for output 'out'; rejecting this build output2025-03-26 05:13:11.974Z: 
2025-03-26 05:13:13.624Z: error: suspicious ownership or permission on '/nix/store/kgxi76yq8fkfg49v3pzixgxr765awzv3-devenv-enterShell' for output 'out'; rejecting this build output
2025-03-26 05:13:14.060Z: error: suspicious ownership or permission on '/nix/store/xsbb6j4jqsr9bzfnscnxc3gy0k8v3i4q-frontend' for output 'out'; rejecting this build output
2025-03-26 05:13:17.293Z: error: suspicious ownership or permission on '/nix/store/0l65lymf8p6366zvqlvfrpbwzj2qrc45-tls.json' for output 'out'; rejecting this build output
2025-03-26 05:13:17.300Z: error: 2 dependencies of derivation '/nix/store/0zl8ycb975riqfwph2j88ps1v6307f84-caddy-config.json.drv' failed to build
error: 1 dependencies of derivation '/nix/store/x9gq8cl4gxfflr5164jqfw1p1hn8gm27-caddy.drv' failed to build
2025-03-26 05:13:29.510Z: error: 6 dependencies of derivation '/nix/store/455561x1n6hvyxfxmaf8fpjrf65908p3-process-compose.yaml.drv' failed to build
2025-03-26 05:15:21.223Z: error:
       error: 4 dependencies of derivation '/nix/store/nxc24dnln4py6c68jvpyslymgb9mjabn-devenv-shell-env.drv' failed to build
2025-03-26 05:15:21.262Z: ✖ Building shell in 173s
2025-03-26 05:15:21.263Z: Error: 2025-03-26 05:15:21.265Z:   x Command `/nix/store/vim5dmixhcrl0aw2svq5nyys4wl1g4gb-nix-
  | devenv-2.24.0pre20241213_bde6a1a/bin/nix --show-trace --extra-
  | experimental-features nix-command --extra-experimental-features
  | flakes --option warn-dirty false --keep-going --max-jobs 2
  | --option eval-cache false --option extra-substituters https://
  | devenv.cachix.org --option extra-trusted-public-keys devenv.cachix.org-
  | 1:w1cLUi8dv3hnoSPGAuibQv+f9TZLr6cv/Hm9XgU50cw= print-dev-env --profile /
2025-03-26 05:15:21.265Z:   | workspaces/2025.ploneconf.org/.devenv/gc/shell -vv --log-format internal-
  | json` failed with with exit code 12025-03-26 05:15:21.267Z: 

2025-03-26 05:15:21.268Z: make: *** [Makefile:229: codespace-install] Error 1
2025-03-26 05:15:21.274Z: updateContentCommand failed with exit code 2. Skipping any further user-provided commands.

2025-03-26 05:15:21.280Z: Error: Command failed: /bin/sh -c make codespace-install
2025-03-26 05:15:21.280Z:     at b7 (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:235:130)
2025-03-26 05:15:21.281Z:     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
2025-03-26 05:15:21.282Z: {"outcome":"error","message":"Command failed: /bin/sh -c make codespace-install","description":"The updateContentCommand in the devcontainer.json failed.","containerId":"470f9c6be7cd5a4355cef34d7a7189e5a9121885d2f81b4e202a16625b7030ef"}
2025-03-26 05:15:21.283Z:     at async Am (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:227:4483)
2025-03-26 05:15:21.283Z:     at async Xw (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:227:3828)
2025-03-26 05:15:21.284Z:     at async $w (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:227:2904)
2025-03-26 05:15:21.284Z:     at async fa (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:227:2438)
2025-03-26 05:15:21.284Z:     at async mtA (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:465:1534)
2025-03-26 05:15:21.286Z:     at async NH (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:465:964)
2025-03-26 05:15:21.286Z:     at async YtA (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:482:3771)
2025-03-26 05:15:21.286Z:     at async eB (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:482:4886)
2025-03-26 05:15:21.289Z: devcontainer process exited with exit code 1

====================================== ERROR ====================================
2025-03-26 05:15:21.297Z: Failed to create container.
=================================================================================
2025-03-26 05:15:21.298Z: Error: Command failed: /bin/sh -c make codespace-install
2025-03-26 05:15:21.313Z: Error code: 1302 (UnifiedContainersErrorFatalCreatingContainer)

====================================== ERROR ====================================
2025-03-26 05:15:21.334Z: Container creation failed.
=================================================================================
2025-03-26 05:15:25.208Z: 

===================================== WARNING ===================================
2025-03-26 05:15:25.210Z: Creating recovery container.
=================================================================================
```

--- a/containers/devcontainer/Dockerfile
+++ b/containers/devcontainer/Dockerfile
@@ -9,7 +9,13 @@ COPY nix.conf /tmp/nix.conf
 
 # Update and install dependencies
 RUN apt-get update && \
-    apt-get install -y bash curl git sudo xz-utils
+    apt-get install -y acl bash curl git sudo xz-utils
+
+# Remove the default permissions from /tmp.
+# This fixes the "suspicious ownership or permission" error from Nix.
+# Requires that the acl package be installed.
+# Upstream issue: https://github.com/NixOS/nix/issues/6680
+RUN sudo setfacl -k /tmp
 
 # Install Nix
 # NOTE: The extra conf file does not apply to the install script itself.

